base_product_merge/readme/DESCRIPTION.md

```diff
@@ -0,0 +1 @@
+We can merge duplicates products into single product
```

Comment:
Maybe `This module allows to merge products that are considered as duplicates into a single product.` is better